melatonin/helpers/misc.h

```diff
@@ -1,36 +1,37 @@
-#pragma once
-#include "../../LatestCompiledAssets/InspectorBinaryData.h"
-#include <juce_audio_processors/juce_audio_processors.h>
-
-namespace melatonin
-{
-    static inline juce::String dimensionsString (juce::Rectangle<int> bounds)
-    {
-        // account for outline border drawing (1 on each side)
-        return juce::String (bounds.getWidth() - 2) + L" × " + juce::String (bounds.getHeight() - 2);
-    }
-
-    static inline juce::String distanceString (juce::Line<float> line)
-    {
-        // account for outline border drawing (1 on each side)
-        auto v = line.getLength() + 3;
-        return juce::String (v);
-    }
-
-    static inline juce::Image getIcon (juce::String iconName)
-    {
-        int size = 0;
-        auto filename = iconName.replace (" ", "") + "_png";
-        auto data = InspectorBinaryData::getNamedResource (filename.getCharPointer(), size);
-        if (size > 0)
-        {
-            return juce::ImageCache::getFromMemory (data, size);
-        }
-        else
-        {
-            // you told me to load an image that doesn't exist
-            jassertfalse;
-        }
-    }
-
-}
```

Comment:
did this file have some line ending issues ? just curious about the diff...

Reply:
Not sure what happened there, maybe xcode changed the line endings? What line ending do you prefer?

Reply:
I have them set to Unix/macOS `\n` but had some trouble in the past, so it's possible it got cleaned up here... github doesn't make it easy to see....